core/azure-core/src/main/java/com/azure/core/polling/PollerOptions.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+/**
+ * Place to provide configuration options for Poller
+ **/
+public class PollerOptions {
+
+    private String negativeValueFormat = "Negative or zero not allowed.Provided value %d";
+    private String timeoutSmallerThanPollIntervalFormat = "Poll Interval %d should be smaller than timeout %d";
+    private String growthFactorValueFormat = "Growth factor %.5f should be equal or larger than 1.0";
+
+    private int timeoutInMilliSeconds;
+    private int pollIntervalInMillis;
+
+    /*This will ensure that poll interval grow exponentially by factor specified here.
+     * This should always be more than 1.*/
+    private float pollIntervalGrowthFactor = 1.0f;
```

Comment:
I don't believe this is required. Exponential growth factor will be calculated by the service, rather than specified by the client library.